webpack.config.js

```diff
@@ -99,6 +95,7 @@ const babelLoader = {
 const webpackConfig = {
 	bail: ! isDevelopment,
 	entry: {},
+	mode: bundleEnv,
```

Comment:
Is this safe? Should we do `bundleEnv === 'development' ? 'development' : 'production'`?

There is a `shared` env that could potentially leak in here:

https://github.com/Automattic/wp-calypso/blob/cd87d90ee7afba0ebc18d301fbc3c0c887a40aca/config/_shared.json#L2